dump/testdata.js

```diff
@@ -79,7 +80,7 @@ db.events.insert({
         rating: 4,
         comment: "Top"
     }],
-    picture: "img/demo/events/man-holding-black-dumbbell-1229356.jpg",
+    picture: "/img/demo/events/man-holding-black-dumbbell-1229356.jpg",
```

Comment:
ich glaube das "/" ist jetzt schon in dem Template mit drinnen, willst du nochmal schauen ob es jetzt passt